third_party/py/python_configure.bzl

```diff
@@ -138,10 +138,8 @@ def _symlink_genrule_for_dir(repository_ctx,
 
 def _get_python_bin(repository_ctx):
     """Gets the python bin path."""
-    python_bin = repository_ctx.os.environ.get(_PYTHON_BIN_PATH)
-    if python_bin != None:
-        return python_bin
-    python_bin_path = repository_ctx.which("python")
+    python_bin = repository_ctx.os.environ.get(_PYTHON_BIN_PATH, 'python')
```

Comment:
Why is this changing the semantics of the `PYTHON_BIN_PATH` environment variable?

Reply:
This library is forked from TensorFlow's repo. The main purpose of this package is locating the CPython header files to help Cython compile. In its original logic, to use Python 3, we need to set `PYTHON_BIN_PATH` to a absolute path like `/usr/bin/python3` or `/usr/local/bin/python3`. It's not very adaptive across environments. So, in the new logic, even if the `PYTHON_BIN_PATH` is not given an absolute path, it can try to find the correct path using `which`.

Also, it doesn't break backward compatibility, developers who used this flag should observe no behavior change, because `which` command will return the exact same path as input if the input is an absolute path.

Reply:
I see that this is used by the `build:python3 --action_env=PYTHON_BIN_PATH=python3` addition in tools/bazel.rc. If the error reported above is indeed valid, I would suggest instead adding a new environment variable (`PYTHON_BIN_NAME`, or something similar, which defaults to `python`) that can be set in `bazel.rc` to `python3`, and pass this value to the existing `which` query.